_footer/footer.md

```diff
@@ -7,16 +7,16 @@ Stay up to date and sign up to our <strong><a href="http://lists.inai.de/iridium
 <br/>
 Please report any issues and address any questions to our <strong><a href="https://github.com/iridium-browser/tracker/issues" title="report issues to Bug Tracker @ GitHub" target="_blank" rel="noreferrer">Bug Tracker @ GitHub</a></strong>
 <br/>
-For more development information please <strong><a href="/development.html" title="more development information">check here</a></strong>
+Get in touch and chat with the team on <strong><a href="https://chat.osb-alliance.de/signup_user_complete/?id=zcw9hio95tf15bttox3jdcsmuc" title="chat with the team on Mattermost" target="_blank" rel="noreferrer">Mattermost</a></strong>
 <br/>
-<span class="fa fa-code"></span> download the <strong><a href="/downloads/source.html" title="download the source code">source code</a></strong>
+<span class="fa fa-code"></span> download the <strong><a href="/downloads/source" title="download the Source Code">Source Code</a></strong>
```

Comment:
please stop misspelling source code all the time. There is no reason for using title case here.

Reply:
...in which case EVERYTHING is misspelled on the entire Internet!

Reply:
Yes, so let's not make it worse.